File: php/squareNSum.php
<?php
function square_sum($numbers): int
{
    $sum = 0;

    for ($i = 0; $i < count($numbers); $i++) {
        $sum += $numbers[$i] * $numbers[$i];
    }

    return $sum;
}

echo square_sum([1, 5]);

File: java/src/SquareNSum.java
public class SquareNSum {
    public static int squareSum(int[] n) {
        int sum = 0;

        for (int i = 0; i < n.length; i++) {
            sum += n[i] * n[i];
        }

        return sum;
    }

    public static void main(String[] args) {
        int[] n = {1, 2, 3, 4, 5};
        System.out.println(SquareNSum.squareSum(n));
    }
}
